### PR TITLE
Add Update Integration endpoint and Web integration

### PR DESCRIPTION
**Pull request by dannytranlx**
Create Integration
image
Update integration
image

### CODE REVIEW

File: slate/source/includes/rest/core/_integrations.md [hunk 1]
```diff
@@ -704,6 +771,73 @@ smooch.integrations.get('55c8d9758590aa1900b9b9f6', '5735dded48011972d621dc02').
...
 Return the specified integration.
 
+## Update Integration
+
+<api>`PUT /v1/apps/{appId}/integrations/{integrationId}`</api>
+
+The Update Integration endpoint allows you to update active integrations. See the sections below for channel specific instructions.
+
+<aside class='notice'>
+The Update Integration endpoint is currently only available for the Web Messenger integration. To update the other integrations, you will need to [Delete Integration](#delete-integration) and [Create Integration](#create-integration) again.
```
Comment (jpjoyal) [inline slate/source/includes/rest/core/_integrations.md]: suggest: "To update other integration types, first delete the integration and re-create it."

File: slate/source/includes/rest/core/_integrations.md [hunk 2]
```diff
@@ -9,6 +9,73 @@ The supported integration types are: Facebook Messenger, Twitter DM, LINE, Teleg
 
 The Create Integration endpoint allows you to provision apps with front-end messaging channels. See the sections below for channel specific instructions.
 
+<h2 id="web-messenger-integration">Web Messenger</h2>
```
Comment (dannytranlx) [inline slate/source/includes/rest/core/_integrations.md]: Another section is already called `Web Messenger` with the anchor `#web-messenger`

File: pages/guide/web-messenger.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ The current supported list of notifiable channels is as follows:
 - Telegram
 - Viber
 
-In order to offer these channels, simply [integrate](https://app.smooch.io/integrations/categories/customer-channels) any you want and they will automatically be added to the list.
+In order to offer these channels, simply [integrate](https://app.smooch.io/integrations/categories/customer-channels) any you want and they will be added to the list by default. You may also edit that list through the [Integration Update endpoint](https://docs.smooch.io/rest/#update-integration).
```
Comment (mspensieri) [inline pages/guide/web-messenger.md]: nit: Update Integration endpoint

same thing below on line 191

File: slate/source/includes/rest/core/_integrations.md [hunk 4]
```diff
@@ -9,6 +9,73 @@ The supported integration types are: Facebook Messenger, Twitter DM, LINE, Teleg
...
+  }
+}
+```
+
+To configure a Web Messenger integration, acquire the required information and call the Create Integration endpoint.
+
+| **Arguments**             |   |
+|---------------------------|---|
+| **type**<br/><span class='req'>required</span> | The integration type: _web_. |
+| **brandColor**<br/><span class='req'>optional</span> | This color will be used in the messenger header and the button or tab in idle state.<br/>Default value is `65758e`. |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_integrations.md]: I would add a note on the format we expect here and for the other colors, something like "must be a three or six digit hexadecimal color code"

File: slate/source/includes/rest/core/_integrations.md [hunk 5]
```diff
@@ -9,6 +9,73 @@ The supported integration types are: Facebook Messenger, Twitter DM, LINE, Teleg
...
+| **conversationColor**<br/><span class='req'>optional</span> | This color will be used for customer messages, quick replies and actions in the footer. <br/>Default value is `0099ff`. |
+| **actionColor**<br/><span class='req'>optional</span> | This color will be used for call-to-actions inside your messages.<br/>Default value is `0099ff`. |
+| **displayStyle**<br/><span class='req'>optional</span> | Choose how the messenger will appear on your website. <br/>Default value is `button`. |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_integrations.md]: we should enumerate the options here, how would they know to pass `tab`?

File: slate/source/includes/rest/core/_integrations.md [hunk 6]
```diff
@@ -9,6 +9,73 @@ The supported integration types are: Facebook Messenger, Twitter DM, LINE, Teleg
...
+| **buttonIconUrl**<br/><span class='req'>optional</span> | With the button style Web Messenger, you have the option of selecting your own button icon. The image must be at least 200 x 200 pixels and must be in either JPG, PNG, or GIF format. |
+| **integrationOrder**<br/><span class='req'>optional</span> | Array of integration IDs, order will be reflected in the Web Messenger. When set, only integrations from this list will be displayed in the Web Messenger. |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_integrations.md]: Suggest mentioning the other behaviour too

> When set, only integrations from this list will be displayed in the Web Messenger. If unset, all integrations will be displayed

File: slate/source/includes/rest/core/_integrations.md [hunk 7]
```diff
@@ -704,6 +771,73 @@ smooch.integrations.get('55c8d9758590aa1900b9b9f6', '5735dded48011972d621dc02').
...
 Return the specified integration.
 
+## Update Integration
+
+<api>`PUT /v1/apps/{appId}/integrations/{integrationId}`</api>
+
+The Update Integration endpoint allows you to update active integrations. See the sections below for channel specific instructions.
+
+<aside class='notice'>
+The Update Integration endpoint is currently only available for the Web Messenger integration. To update other integration types, first delete the integration and re-create it.
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_integrations.md]: alternate wording suggestion

> To update other integration types, you must instead delete the integration and re-create it with the new data.

File: slate/source/includes/rest/core/_integrations.md (hunk 4 above)
Comment (dannytranlx) [reply slate/source/includes/rest/core/_integrations.md]: We don't do any validations on the color tho, but yeah was surprised 3-hex code works, i'll add something, good point!

**Pull request merged**